src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -141,6 +141,9 @@
   <data name="SizeLabel" xml:space="preserve">
     <value>Size:</value>
   </data>
+  <data name="DimensionsLabel" xml:space="preserve">
+    <value>Dimensions:</value>
```

Comment:
There is already a string for dimensions (see the details pane). It doesn't have a colon at the end but I think we can reuse the string and just insert the colon from the code.

Reply:
Thank you